lib/ClangImporter/ClangImporter.cpp

```diff
@@ -68,7 +68,9 @@ using namespace importer;
 using clang::CompilerInstance;
 using clang::CompilerInvocation;
 
+#if !defined(_MSC_VER)
 #pragma mark Internal data structures
+#endif
```

Comment:
It reports warnings. The reason why I didn't do `if defined(__clang__)` is because MSVC is the only guy out of the lot not to simply ignore